qiskit/pulse/pulse_lib/continuous.py

```diff
@@ -42,42 +42,42 @@ def zero(times: np.ndarray) -> np.ndarray:
     return constant(times, 0)
 
 
-def square(times: np.ndarray, amp: complex, period: float, phase: float = 0) -> np.ndarray:
+def square(times: np.ndarray, amp: complex, freq: float, phase: float = 0) -> np.ndarray:
```

Comment:
Do you think we need to add a warning here @lcapelluto? I don't think these pulses have been seeing much use so far.

Reply:
yeah, I don't think so, either, but I don't see any reason not to add the warning. I'd say best to assume someone may be using this